lib/node_modules/@stdlib/blas/ext/base/ndarray/dlast-index-of/lib/main.js

```diff
@@ -78,14 +78,11 @@ function dlastIndexOf( arrays ) {
 	} else if ( fromIndex >= N ) {
 		return -1;
```

Comment:
@headlessNode This behavior does not match `Array.prototype.lastIndexOf` (see https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/lastIndexOf#fromindex). Namely, when `fromIndex` exceeds the last index `N-1`, the `fromIndex` is clamped to the last index. We should probably follow similarly here. We should also check that `index-of` semantics more or less align with `Array.prototype.indexOf`.